### PR TITLE
Update httpure

### PR DESCRIPTION
**Pull request by sigma-andex**

### CODE REVIEW

**Review by cprussin — APPROVED** [review]

**Comment by cprussin** [comment]
thanks @sigma-andex

File: src/groups/cprussin.dhall [hunk 1]
```diff
@@ -53,7 +53,43 @@
...
+    , "refs"
+    , "spec"
+    , "strings"
+    , "tuples"
+    , "type-equality"
+    , "unsafe-coerce"
+    ]
+  , repo = "https://github.com/cprussin/purescript-httpure.git"
+  , version = "v0.11.0"
+  }
```
Comment (thomashoneyman) [inline src/groups/cprussin.dhall:91]: Could you remove the commented-out definition for `httpure` above?

Also, while the previous version already had it, it looks like there are some unnecessary dependencies in here. We only need production dependencies in package sets. `psci-support` and `spec` could most likely be dropped, and maybe even `console` if it isn't imported anywhere.

Comment (sigma-andex) [reply src/groups/cprussin.dhall:91]: Yeah will have a look at it. I think there is also still something broken. Will have a look at it tomorrow

Comment (thomashoneyman) [reply src/groups/cprussin.dhall:91]: Looks like you're missing the `js-uri` dependency

**Comment by sigma-andex** [comment]
Ok I think I finally got it. Thanks a lot for the explanations @cprussin  and @thomashoneyman  :-)

**Pull request merged**